deploy_to_kubernetes_cron.sh

```diff
@@ -29,7 +29,7 @@ for command in "${!commands[@]}"; do
   echo SCHEDULE "$SCHEDULE"
   echo PRIMARY_IMAGE "$PRIMARY_IMAGE"
 
-  kubectl delete cronjob $NAME
+  kubectl delete cronjob $NAME || true
```

Comment:
I think it would be better to conditionally execute this instead of executing it with a bogus argument.

Reply:
why?

Reply:
talked offline, this is somewhat of a standard bash thing. the output of the command will go to the pipe true and cause the script to get past the line if it errors. sounds good to me. 👍